org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/outline/quickoutline/StringMatcher.java

```diff
@@ -60,7 +61,11 @@ protected Pattern getPattern() {
 	}
 	
 	public boolean match(String text) {
-		return getPattern().matcher(text).find();
+		try {
+			return getPattern().matcher(text).find();
```

Comment:
Would it make sense to use Pattern.quote(..) and in translatePattern(..) and add a special handling for * and . ?

Reply:
am not sure if i am firm enhough with patterns to do this

Reply:
:) The catch does certainly no harm.